.github/workflows/test_build.yml

```diff
@@ -1,6 +1,9 @@
 name: Build/Push Nightly Binary
 
 on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 * * *' 
```

Comment:
add what this means to the comment => every day at x pm etc

Reply:
Those can be looked up on Github docs: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule